packages/dbos-cloud/cloudutils.ts

```diff
@@ -78,11 +80,23 @@ export function getCloudCredentials(): DBOSCloudCredentials {
   const userCredentials = JSON.parse(fs.readFileSync(`./${dbosEnvPath}/credentials`).toString("utf-8")) as DBOSCloudCredentials;
   const credentials =  {
     userName: userCredentials.userName,
+    refreshToken: userCredentials.refreshToken,
     token: userCredentials.token.replace(/\r|\n/g, ""), // Trim the trailing /r /n.
   };
   if (isTokenExpired(credentials.token)) {
-    logger.error("Error: Login expired. Please log in again with 'npx dbos-cloud login'")
-    process.exit(1)
+    if (credentials.refreshToken) {
+      logger.info("Refreshing access token with refresh token")
+      const authResponse = await authenticateWithRefreshToken(logger, credentials.refreshToken);
+      if (authResponse === null) {
+        process.exit(1);
```

Comment:
Print an error message here before exit. Maybe "Please log out with `npx dbos-cloud logout` and log in again".

Reply:
Can we even just logout automatically here and prompt users to login again?

Reply:
Fixed!